nameserver/nameserver.go

```diff
@@ -176,6 +281,23 @@ func (n *Nameserver) deleteTombstones() {
 	n.entries.filter(func(e *Entry) bool {
 		return e.Tombstone == 0 || now-e.Tombstone <= int64(tombstoneTimeout/time.Second)
```

Comment:
I addressed that elsewhere - clear the `stopped` flag on `ContainerDestroyed`.
Do we want the behaviour to depend on how long the container is stopped for?
I guess we do something similar in IPAM.